src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns="http://java.sun.com/xml/ns/javaee"
+	      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	      xsi:schemaLocation="http://java.sun.com/xml/ns/javaee 
+	      http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
+	      version="2.5">
+
+  <display-name>canvas_courses_report</display-name>
+  
+    <servlet>
+		<servlet-name>AnalyticsServlet</servlet-name>
```

Comment:
I think analytics is not the best choice since this only does a tiny bit of analytics.  Might as well not be ctools either, since this has little to do with ctools.  Maybe "tl" instead of "ctools"?